app/notifications/push/index.ts

```diff
@@ -0,0 +1,76 @@
+import EJSON from 'ejson';
+
+import store from '../../lib/createStore';
+import { deepLinkingOpen } from '../../actions/deepLinking';
+import { isFDroidBuild } from '../../constants/environment';
+import PushNotification from './push';
+
+export interface INotification {
+	message: string;
+	style: string;
+	ejson: string;
+	collapse_key: string;
+	notId: string;
+	msgcnt: string;
+	title: string;
+	from: string;
+	image: string;
+	soundname: string;
+	// only for info
+	'google.delivered_priority': string;
```

Comment:
If we don't use this google's fields in other places, I think we don't need maintain here.